generator/operation.go

```diff
@@ -709,6 +709,10 @@ func (b *codeGenOpBuilder) MakeHeader(receiver, name string, hdr spec.Header) (G
 	}
 
 	if hdr.Items != nil {
+		if res.CollectionFormat == "" {
+			// When not defined,CollectionFormat defaults to csv
+			res.CollectionFormat = "csv"
```

Comment:
not a fan of this because it now sprinkles this across the codebase and people have to remember to do this. Better to postpone this until the last responsible moment

Reply:
Need some guidance as I am not following you on this 
This was supposed to be done by spec (<= the first possible moment) and it is not.

Last possible moment is render time. Don't you think we haven't already loaded the templates enough with logics like this?

Reply:
"" and "csv" should be functionally equivalent. The runtime, strfmt and swag functions that deal with this should equate "" and "csv".

Reply:
OK